README.md

```diff
@@ -212,11 +212,21 @@ vim packages/backend/config/env/production.json
       "email": "@@DEFAULT_ADMIN_EMAIL",
       "password": "@@DEFAULT_ADMIN_PASSWORD"
     },
-    "sendgrid": {
-      "senderEmail": "@@SENDGRID_SENDER_EMAIL",
-      "confirmationTemplateId": "@@SENDGRID_CONFIRMATION_TEMPLATE",
-      "resetPasswordTemplateId": "@@SENDGRID_RESET_PASSWORD_TEMPLATE",
-      "apiKey": "@@SENDGRID_API_KEY"
+    "mail": {
```

Comment:
Let's leave the old sendgrid configuration block as well for backwards compatibility (and remove it in 1-2 versions). Let's also display a warning if the old config format is used.